plugins/linux-capture/pipewire.c

```diff
@@ -578,6 +578,11 @@ static void on_param_changed_cb(void *user_data, uint32_t id,
 
 	spa_format_video_raw_parse(param, &obs_pw->format.info.raw);
 
+	// Announce support for DMA-BUF only if negotiation results in a modifier
+	if (check_pw_version(&obs_pw->server_version, 0, 3, 24)) {
+		buffertypes |= 1 << SPA_DATA_DmaBuf;
+	}
```

Comment:
A few comments here:

  1. Seems like the `buffertypes` variable doesn't exist, nor it used, in this function
  2. Could you please rename it to `buffer_types`
  3. Please remove the curly braces

Reply:
Thanks, lost on rebase.